### DISCUSSION TITLE
Buy event

### DISCUSSION THREAD

**Discussion by FewerElk** [ScreamingSandals/BedWars#490] [Q&A] [answered]
Hi, I'm coding an extension for this plugin for the issue https://github.com/ScreamingSandals/BedWars/issues/488, and is there a specific event for when a player buy an item ? Else I will try with some basic api, but I think it is not a good idea.

**Comment by zlataovce** [comment, marked as answer]
https://jd.screamingsandals.org/sbw-0-2-x/BedWars-API/org/screamingsandals/bedwars/api/events/BedwarsItemBoughtEvent.html

**Reply by FewerElk** [reply]
Does the basic import work with it class (I practise the Java language from 2 month)

**Reply by FewerElk** [reply]
Does the basic import work with it class (I practise the Java language from 2 month)

**Reply by zlataovce** [reply]
you can import it and use it as a normal Bukkit event as long as you have the BedWars API dependency (https://docs.screamingsandals.org/BedWars/0.2.28/api/)
